applications/luci-app-frpc/htdocs/luci-static/resources/view/frpc.js

```diff
@@ -115,12 +116,68 @@ function defOpts(s, opts, params) {
 	}
 }
 
+var conf = 'frpc';
+var callServiceList = rpc.declare({
+	object: 'service',
+	method: 'list',
+	params: ['name'],
+	expect: { '': {} }
+});
+
+function getPid() {
+	return L.resolveDefault(callServiceList(conf), {}).then(function (res) {
+		var isrunning = false;
+		try {
+			isrunning = res[conf]['instances']['instance1']['running'];
+		} catch (e) { }
+		return isrunning;
+	});
+}
+
+function getServiceStatus() {
+	return Promise.all([
+		getPid()
+	]);
+}
+
+function renderStatus(res) {
+	var renderHTML = "";
+	var isRunning = res[0];
+
+	if (isRunning) {
+		renderHTML += "<span style=\"color:green;font-weight:bold\">&nbsp;&nbsp;&nbsp;&nbsp;" + _("frp Client") + " - " + _("RUNNING") + "</span>";
+	} else {
+		renderHTML += "<span style=\"color:red;font-weight:bold\">&nbsp;&nbsp;&nbsp;&nbsp;" + _("frp Client") + " - "  + _("NOT RUNNING") + "</span>";
+		return renderHTML;
```

Comment:
Not needed. Only one return statement at function end

Reply:
a mistake.